.github/workflows/solidity-tests.yml

```diff
@@ -46,6 +46,13 @@ jobs:
     with:
       testfilter: '@solidityequiv3'
 
+  OpenZeppelinEquivalence:
```

Comment:
Nit.  Are we defining an OpenZeppelinEquivalence?  Perhaps just OpenZeppelin?

Reply:
Yeah you're right that makes more sense! Just pushed the fix in!